bin/runtime/src/lib.rs

```diff
@@ -433,6 +495,7 @@ construct_runtime!(
         Aleph: pallet_aleph::{Pallet, Call, Config<T>, Storage, Event<T>},
         Session: pallet_session::{Pallet, Call, Storage, Event, Config<T>},
         Scheduler: pallet_scheduler::{Pallet, Call, Storage, Event<T>},
+		Treasury: pallet_treasury::{Pallet, Call, Storage, Config, Event<T>},
```

Comment:
same, weird formatting (but seems to be passing fmt checks :thinking: )

Reply:
It's because this is macro code. Fmt does not have power there :P

Reply:
changed tabs to spaces (must have been introduced while copying from substrate :|)